addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -507,7 +507,7 @@ export class WebsitePreview extends Component {
         ev.preventDefault();
         const path = this.websiteService.contentWindow.location;
         const debugMode = this.env.debug ? `?debug=${odoo.debug}` : "";
```

Comment:
The different references are an oversight on my part, I forgot to change `odoo.debug` inside the string.

However, I don't think it's as simple as doing `?debug=true` since we also have the `debug=assets` mode. 😄 

I pushed a change to not use `odoo.debug`